components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/inbound/IdentityServlet.java

```diff
@@ -107,7 +108,11 @@ private HttpIdentityResponse process(HttpServletRequest request, HttpServletResp
             }
             return responseBuilder.build();
         } catch (FrameworkException e) {
-            log.error("Failed to process IdentityRequest", e);
+            if (e instanceof SessionExtenderClientException) {
+                log.debug("Failed to process IdentityRequest", e);
```

Comment:
let's check if debug is enabled before adding debug. As that is the general format we use.